provisioner/plain-v0/README.md

```diff
@@ -1,3 +1,197 @@
-# plain-v0
+# Plain-v0 Provisioner
 
-A concrete Provisioner implementation for "plain" k8s bundles.
+## Summary
+
+The `plain-v0` provisioner is a core rukpak provisioner that knows how to interact with `plain+v0` format bundles.
```

Comment:
It seems confusing to me that most things use `plain-v0`, but we're calling the format `plain+v0`. Should we make it `plain-v0` everywhere for consistency?

Reply:
From what I understood we were doing that intentionally to separate between the provisioner type and the bundle format. Since we have the existing `registry+v1` format, calling this format `plain+v0` seems consistent.